svelte/src/routes/v2/scripts/draw/index.ts

```diff
@@ -312,6 +313,7 @@ export function draw(
 
 	// Add zoom handler
 	svg.call(
+		//@ts-ignore
 		d3
```

Comment:
Hmm yeah kinda weird that it doesn't recognize SVGElement as Element even though it extends Element. I tried asserting it into `(svg as d3.Selection<SVGElement | Element, unknown, any, any>)` but I'm not sure if this is a good practice. I was just hoping it would help in the future.